_includes/manuals/1.14/4.4.4_prov_ptables.md

```diff
@@ -3,6 +3,142 @@ Partition templates are a subset of normal Provisioning Templates. They are hand
 
+#### Per-Host Partition tables
+
+When creating a new Host, you will be given the option to create an individual partition table. This is essentially a 'one-off' partition table that is stored with the host and used only for that host. It replaces the choice of Partition Table from the normal list of those associated with the selected OS.
+
 #### Dynamic Partition tables
+Some operating systems allow you to create partition tables via scripts. At the moment Kickstart and AutoYaST based systems can use this feature. Partition templates starting with ```#Dynamic``` are interpreted as scripts rather than static partition tables. The Provisioning Template needs to support this feature (search for ```@dynamic```). This enables you to make choices on the fly during provisioning (or re-provisioning).
+
+##### Kickstart Dynamic Partition tables
+Kickstart will run dynamic partition tables as a pre-install bash script using a %pre scriplet. This script needs to create a complete Kickstart partition table in '/tmp/diskpart.cfg'. This partition table will then be read by anaconda for the installation by using ```%include /tmp/diskpart.cfg```.
+
+Example Dynamic Partition table:
+```
+#Dynamic - this line tells Foreman this is a script rather then a static layout
+#This snippets define the swap partition size, it would generate a partition twice the size of the memory if your physical memory is up to 2GB
+#or will create a swap partition with your memory size + 2GB.
+
+#get the actual memory installed on the system and divide by 1024 to get it in MB
+act_mem=$((`grep MemTotal: /proc/meminfo | sed 's/^MemTotal: *//'|sed 's/ .*//'` / 1024))
+
+#check if the memory is less than 2GB then swap is double the memory else it is memory plus 2 GB
+if [ "$act_mem" -gt 2048 ]; then
+    vir_mem=$(($act_mem + 2048))
+else
+    vir_mem=$(($act_mem * 2))
+fi
+
+#copy all the HDD partitions to the temp file for execution
+cat <<EOF > /tmp/diskpart.cfg
+zerombr yes
+clearpart --all --initlabel
+part swap --size "$vir_mem" 
+part /boot --fstype ext3 --size 100 --asprimary
+part / --fstype ext3 --size 1024 --grow
+EOF
+```
+
+##### AutoYaST Dynamic Partition tables
+AutoYaST will run dynamic partition tables as a pre-install bash script. This script needs to create a AutoYaST XML file in ```/tmp/profile/modified.xml```. The modified.xml file will be read by YaST after your script has finished. An example for getting the same functionality as with Kickstart would be to create your XML partition table in ```/tmp/diskpart.cfg``` and sed it together with the original AutoYaST XML like this: ```sed '/<\/ntp-client>/ r /tmp/diskpart.cfg' /tmp/profile/autoinst.xml > /tmp/profile/modified.xml```. Inserting after ntp-client section is just a suggestion that uses the same style the community-tamplates do. The following example is based on ```disklayout_lvm.erb```.
```

Comment:
1. "tamplates" -> "templates"
1. Not sure the line about this being a suggestion is necessary.
1. "disklayout_lvm.erb" is an internal filename within the codebase, I don't think it's helpful to reference it. Either use the name or remove that comment.